MaeParser.cpp

```diff
@@ -520,7 +520,9 @@ void IndexedBlockBuffer::value(Buffer& buffer)
     }
 }
 
-DirectIndexedBlockParser::~DirectIndexedBlockParser() {}
+DirectIndexedBlockParser::~DirectIndexedBlockParser()
```

Comment:
If possible, please delete these default destructors.

Reply:
Ok. I already checked, and that seems to solve the issue.